pallets/remote-asset-manager/src/calls/proxy.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright 2021 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//! Xcm support for `pallet_proxy` calls
+use codec::{Decode, Encode, Output};
+use frame_support::{sp_std::vec::Vec, weights::Weight, RuntimeDebug};
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+use crate::{CallEncoder, EncodeWith, PalletCall, PalletCallEncoder};
+
+/// The index of `pallet_proxy` in the polkadot runtime
+pub const POLKADOT_PALLET_PROXY_INDEX: u8 = 29u8;
+
+/// The identifier the `ProxyType::Staking` variant encodes to
+pub const POLKADOT_PALLET_PROXY_TYPE_STAKING_INDEX: u8 = 3u8;
```

Comment:
Make these configurable?

Reply:
These are fixed values, 

https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L1021

https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L849

They will be required when adding a ProxyConfig and StakingConfig to PINT at genesis